src/test/kotlin/fi/metatavu/timebank/api/test/functional/tests/TimeEntriesTest.kt

```diff
@@ -55,4 +61,32 @@ class TimeEntriesTest: AbstractTest() {
             }
         }
     }
+
+    /**
+     * Tests /v1/ForecastTimeEntriesDelete/webhook endpoint
```

Comment:
This does sound like a weird path. Is it the same where the application is actually handling these requests?

Reply:
Yes this is incorrect the correct path is /v1/ForecastTimeEntriesDeleteWebhook